### PR TITLE
only fetch active users for github and banner

### PR DESCRIPTION
**Pull request by asood123**

### CODE REVIEW

**Comment by asood123** [comment]
Requires https://github.com/OpenCollective/opencollective-api/pull/542 to be merged first.

File: server/middlewares.js [hunk 1]
```diff
@@ -8,10 +8,10 @@ import filterCollection from '../frontend/src/lib/filter_collection';
 /**
  * Fetch users by slug
  */
-const fetchUsers = (options) => {
+const fetchActiveUsers = (options) => {
   return (req, res, next) => {
     api
-      .get(`/groups/${req.params.slug}/users`, options)
+      .get(`/groups/${req.params.slug}/active_users`, options)
```
Comment (sedubois) [inline server/middlewares.js]: I'd suggest to add an option to the `users` endpoint instead of introducing new one

Comment (asood123) [inline server/middlewares.js]: done.

**Comment by asood123** [comment]
Merging this since seb reviewed this already.

**Pull request merged**